configuration/src/main/java/io/camunda/configuration/SecondaryStorageDatabase.java

```diff
@@ -15,6 +15,9 @@ public abstract class SecondaryStorageDatabase {
   /** Endpoint for the database configured as secondary storage. */
   private String url = "http://localhost:9200";
 
+  /** Prefix to apply to the indexes */
+  private String indexPrefix;
```

Comment:
:x: This requires a default, I assume. The default in ConnectConfiguration in `""`. So we might as well set it here.

Reply:
True, I forgot about this!
Nice catch, thanks!